workers/generic-worker/gwconfig/config_windows.go

```diff
@@ -3,12 +3,14 @@ package gwconfig
 import "testing"
 
 type PublicPlatformConfig struct {
+	EnableInteractive        bool `json:"enableInteractive"`
```

Comment:
This can now live in the global `PublicConfig` and set the default in `loadConfig()`. We'll need to clean up the other files that separate out this flag based on the platform.

Looks like the following changes would be needed here:
* Removed from `workers/generic-worker/gwconfig/config_linux.go`
* I believe `workers/generic-worker/gwconfig/config_other.go` can entirely be removed